packages/okta-auth-js/lib/builderUtil.js

```diff
@@ -13,8 +13,10 @@
 var AuthSdkError = require('./errors/AuthSdkError');
 var tx = require('./tx');
 var util = require('./util');
+var SDK_VERSION = require('../package.json').version;
```

Comment:
If we require package.json during webpack we will end up with a lot of information in our webpack bundle which we don't want there. We should only require package.json if we are sure we are running server side. This file (builderUtil) is used by both webpack and server side code paths. Within webpack we have it set as a global variable. The node-side serverIndex could do something similar, read from package json and set as global variable.

Reply:
Gotcha, then I can pass `SDK_VERSION` as an argument in getUserAgent.